components/taxonomy/src/services.rs

```diff
@@ -209,6 +209,16 @@ pub enum ChannelKind {
     /// ```
     DoorLocked,
 
+    /// This kind is used with values that need to specify whether they're in secure mode.
+    /// ```
+    /// use foxbox_taxonomy::services::*;
+    /// use foxbox_taxonomy::parse::*;
+    ///
+    /// let parsed = ChannelKind::from_str("\"Secure\"").unwrap();
+    /// assert_eq!(parsed, ChannelKind::Secure);
+    /// ```
+    Secure,
+
```

Comment:
I realize we likely want something like Pair or Pairing here. Secure is the Type, not the Kind...

I think you'll have to do the change and ask a review from yoric on monday...